#!/usr/bin/env python
PACKAGE = "s4_vive"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("pos_x", double_t, 0, "pos_x", 0, -10.0, 10.0)
gen.add("pos_y", double_t, 0, "pos_y", 0, -10.0, 10.0)
gen.add("pos_z", double_t, 0, "pos_z", 0, -10.0, 10.0)
gen.add("roll",  double_t, 0, "roll",  0, -3.14, 3.14)
gen.add("pitch", double_t, 0, "pitch", 0, -3.14, 3.14)
gen.add("yaw",   double_t, 0, "yaw",   0, -3.14, 3.14)

exit(gen.generate(PACKAGE, "s4_vive", "transform_rpy"))
